app.js

```diff
@@ -0,0 +1,54 @@
+var cookieParser = require("cookie-parser");
+var createError = require("http-errors");
+var express = require("express");
+var env = require("dotenv");
+var logger = require("morgan");
+var mongoose = require('mongoose');
+var path = require("path");
+
+var indexRouter = require("./routes/index");
+var usersRouter = require("./routes/users");
+var catalogRouter = require("./routes/catalog");
+
+var app = express();
+
+env.config();
+
+const dev_db_url =
+`mongodb+srv://Aniket:${process.env.password}@cluster0.9voqa.mongodb.net/${process.env.dbname}?retryWrites=true&w=majority`;
```

Comment:
Please store the string in the env file.

Reply:
ok

Reply:
Have you made the change? I do not see it here.

Reply:
yes i had store in the env file